File: src/HolzShots/UI/BackgroundForm.vb
Namespace UI
    Friend Class BackgroundForm
        Inherits Form

        Sub New(location As Point, size As Size)
            StartPosition = FormStartPosition.Manual
            FormBorderStyle = FormBorderStyle.None
            BackColor = Color.Black
            Me.Size = size
            Me.Location = location
            ShowInTaskbar = False
        End Sub
    End Class
End Namespace

File: src/HolzShots.Windows/Forms/BackgroundForm.cs
using System.Drawing;

namespace HolzShots.Windows.Forms;

public class BackgroundForm : System.Windows.Forms.Form
{
    public BackgroundForm(Point location, Size size)
    {
        StartPosition = System.Windows.Forms.FormStartPosition.Manual;
        FormBorderStyle = System.Windows.Forms.FormBorderStyle.None;
        BackColor = Color.Black;
        Size = size;
        Location = location;
        ShowInTaskbar = false;
    }
}
